src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/IntegrationTestBase.kt

```diff
@@ -135,24 +134,43 @@ abstract class IntegrationTestBase {
 
   protected fun noActiveInductionResponse(crn: String) {
     val inductionRequest =
-      HttpRequest.request().withPath("/offenders/crn/$crn/contact-summary/inductions").withMethod("GET")
+      request().withPath("/offenders/crn/$crn/contact-summary/inductions")
 
     communityApi.`when`(inductionRequest, Times.exactly(1)).respond(
       response().withContentType(APPLICATION_JSON).withBody("[]")
     )
   }
 
   protected fun tierCalculationResponse(crn: String) {
-    val request = HttpRequest.request().withPath("/crn/$crn/tier")
+    val request = request().withPath("/crn/$crn/tier")
     hmppsTier.`when`(request).respond(
       response().withContentType(APPLICATION_JSON).withBody("{\"tierScore\":\"B3\"}")
     )
   }
 
+  protected fun twoActiveConvictionsResponse(crn: String) {
+    val convictionsRequest =
+      request().withPath("/offenders/crn/$crn/convictions")
+
+    communityApi.`when`(convictionsRequest, Times.exactly(1)).respond(
+      response().withContentType(APPLICATION_JSON).withBody(twoActiveConvictionsResponse())
+    )
+  }
+
+  protected fun singleActiveAndInactiveConvictionsResponse(crn: String) {
+    val convictionsRequest =
+      request().withPath("/offenders/crn/$crn/convictions")
+    communityApi.`when`(convictionsRequest, Times.exactly(1)).respond(
+      response().withContentType(APPLICATION_JSON).withBody(singleActiveAndInactiveConvictionsResponse())
+    )
+  }
+
   protected fun allDeliusResponses(crn: String) {
     singleActiveConvictionResponse(crn)
     singleActiveInductionResponse(crn)
     offenderSummaryResponse(crn)
     tierCalculationResponse(crn)
+    singleActiveConvictionResponse(crn)
```

Comment:
this mock has already been setup on line 169, is it required?

Reply:
there are now up to 3 requests for convictions

Reply:
I could make the setup clearer though, as the first is for active convictions only